### PR TITLE
⚡ Fix keybr.com (implement)

### PR DESCRIPTION
**Pull request by PiggyPlex**
> Refactoring and fixed for CI/CD. Unsure about the initial bug.
> Please let me know if anyone experiences issues with the presence!

`Referencing` #3090 #2996 

**Screenshots**
image
image
image
image

### CODE REVIEW

File: websites/K/keybr.com/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        "id": "695608126602477629"
+    },
+    "url": "www.keybr.com",
+    "description": {
+        "en": "Teaching the world to type at the speed of thought! Typing lessons that work."
+    },
+    "service": "keybr.com",
+    "version": "1.0.1",
+    "logo": "https://i.imgur.com/5ZUdx7o.png",
+    "thumbnail": "https://i.imgur.com/D6gTvpU.png",
```
Comment (callumok2004) [inline websites/K/keybr.com/dist/metadata.json:14]: Must be 512x512

File: websites/K/keybr.com/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    "author": {
+        "name": "PiggyPlex",
+        "id": "695608126602477629"
+    },
+    "url": "www.keybr.com",
+    "description": {
+        "en": "Teaching the world to type at the speed of thought! Typing lessons that work."
+    },
+    "service": "keybr.com",
+    "version": "1.0.1",
```
Comment (callumok2004) [inline websites/K/keybr.com/dist/metadata.json:12]: Change to 1.0.0 since its new

File: websites/K/keybr.com/presence.ts [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+      'privacy-policy': 'Privacy Policy'
+    };
+    presenceData = {
+      ...presenceData,
+      details: 'Viewing page:',
+      state: pages[window.location.pathname?.slice(1)?.toLowerCase()] || 'Unknown Page'
+    };
+  };
+  presence.setActivity(<PresenceData>presenceData, true);
+  presence.setTrayTitle();
```
Comment (callumok2004) [inline websites/K/keybr.com/presence.ts:171]: Only set tray title of there is no presenceData to send (state/details)

**Review by callumok2004 — CHANGES_REQUESTED** [review]

File: websites/K/keybr.com/presence.ts [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+        errors,
+        errorIncrease,
+        score,
+        scoreIncrease,
+        confidence,
+        learningRate,
+        learningKey,
+        learningKeys,
+        __pageData
+      });
```
Comment (Bas950) [inline websites/K/keybr.com/presence.ts:92]: remove console log

**Review by Bas950 — CHANGES_REQUESTED** [review]
And all things callum said

File: websites/K/keybr.com/presence.ts [hunk 5]
```diff
@@ -0,0 +1,172 @@
+const presence: Presence = new Presence({
+  clientId: '812689086435164251'
+}), startTime: number = new Date().getTime();
```
Comment (ririxidev) [inline websites/K/keybr.com/presence.ts:3]: It should be
```javascript
startTime = Math.floor(Date.now() / 1000)
```

File: websites/K/keybr.com/dist/metadata.json [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+        "keybr"
+    ],
+    "category": "games",
+    "settings": [
+        {
+            "id": "changedetails",
+            "title": "Change top line",
+            "icon": "fas fa-trophy",
+            "value": "Current Key: %currentkey%",
+            "placeholder": "Current Key: %currentkey%"
```
Comment (ririxidev) [inline websites/K/keybr.com/dist/metadata.json:28]: Actually, placeholder isn't necessary.

File: websites/K/keybr.com/dist/metadata.json [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+        },
+        {
+            "id": "changestate",
+            "title": "Change second line",
+            "icon": "fas fa-info",
+            "value": "WPM: %speed.wpm% | Score: %score%",
+            "placeholder": "WPM: %speed.wpm% | Score: %score%"
```
Comment (ririxidev) [inline websites/K/keybr.com/dist/metadata.json:35]: Actually, placeholder isn't necessary.

**Review by ririxidev — CHANGES_REQUESTED** [review]
Heya @PiggyPlex!
You should check out thing(s) that I requested to change in my review and other changes requested by @Bas950 & @callumok2004. Please look at them!

P.S. Please don't make your code like it was taken from some minifier because for us, it only makes harder to read your code and it will take long to accept it. so please next time name these functions normally instead of ``d, a`` etc. and make less functions since they are not necessary. Thank youu!!!

Thanks,
**ririxi**. PreMiD Reviewer Team.
Contact: mail@ririxi.dev, ririxi#2721

**Comment by ririxidev** [comment]
Heya @PiggyPlex! We have some news for you that you should take look at:
We are so sorry but we need to deny your pull request regarding this presence due of:
 - No response since first review (7 days as specified in guidelines).
 
Thanks,
**ririxi**. PreMiD Reviewer Team.
Contact: mail@ririxi.dev, ririxi#2721

**Pull request closed without merging**